src/test/java/org/broadinstitute/hellbender/tools/GenomicsDBImportIntegrationTest.java

```diff
@@ -55,39 +55,109 @@ public void testGenomicsDBImportGCSInputs() throws IOException {
     testGenomicsDBImporter(Arrays.asList(hg00096_cloud, hg00268_cloud, na19625_cloud), interval, combined);
   }
 
+  @Test
+  public void testGenomicsDBImportFileInputsInBatches() throws IOException {
+    final String hg00096 = largeFileTestDir + "gvcfs/HG00096.g.vcf.gz";
+    final String hg00268 = largeFileTestDir + "gvcfs/HG00268.g.vcf.gz";
+    final String na19625 = largeFileTestDir + "gvcfs/NA19625.g.vcf.gz";
+    final String combined = largeFileTestDir + "gvcfs/combined.gatk3.7.g.vcf.gz";
+    final SimpleInterval interval = new SimpleInterval("chr20", 17960187, 17981445);
+
+    testGenomicsDBImporterWithBatchSize(Arrays.asList(hg00096, hg00268, na19625), interval, combined);
+  }
+
+  @Test(groups = {"bucket"})
+  public void testGenomicsDBImportGCSInputsInBatches() throws IOException {
```

Comment:
Recommend that you test with several different batch sizes here (1, 2, 3, and I'd also include 4 to prove that the tool works even if you pass in a batch size > the number of samples). You can easily do this by adding a `DataProvider` for this method that provides the batch sizes, and then pass the batch size into `testGenomicsDBImporterWithBatchSize()` on each invocation.